src/system/packages/brew/shim.rb

```
@@ -0,0 +1,885 @@
+    @sha256 = sha unless sha.nil?
+    @sha256
+  end
+
+  def version(version = nil)
+    @version = version unless version.nil?
+    @version || (@url && Version.new(@url[/[0-9]+(?:\.[0-9]+)+/].to_s))
+  end
+
+  def mirror(url) = (@mirrors ||= []) << url
+  def using = @specs[:using]
+  def livecheck(&) = nil
```

RuboCop (1.87.0)

[fatal] unexpected token tRPAREN
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)